src/containers/admin/dashboard.js

```diff
@@ -47,17 +47,7 @@ class Dashboard extends React.Component {
     return (
       <div>
         <h1>Dashboard</h1>
-        {USERS.map((user, i) => (
-          <Row key={i}>
-            {Object.keys(user).map((key, i) => (
-              <p key={i} style={{ flex: "1" }}>
-                {user[key]}
-              </p>
-            ))}
-          </Row>
-        ))}
-
-        <Grid columns={columnData} data={dataArr} />
+        <ApplicationsTable columns={columnArr} data={dataArr} />
```

Comment:
since we are using Redux, this passing of props is unnecessary here and we can make `<ApplicationsTable>` itself intelligent enough to fetch the data we need. So for the time being, just put the `columnArr` variable within the `<ApplicationsTable>` component itself.

Reply:
makes sense; good idea